packages/vx-demo/src/components/PackageList.tsx

```diff
@@ -19,43 +20,67 @@ export default function PackageList({
           <HeaderElement>Chart primitives</HeaderElement>
           <ul>
             <li className={cx(emphasizePackage === 'annotation' && 'emphasize')}>
-              <a href="/docs/annotation">@vx/annotation</a>
+              <Link href="/docs/annotation">
```

Comment:
all these are the same, just wrapped `<a />`s in `<Link />`s (without the `<a />`s, `next` was complaining about direct string children being deprecated and suggested adding `<a />`s as a child of `<Link />`)